runtime/oti/j9nonbuilder.h

```diff
@@ -1708,6 +1713,8 @@ typedef struct J9FlattenedClassCache {
 
 #define J9_VM_FCC_ENTRY_FROM_FCC(flattenedClassCache, index) (((J9FlattenedClassCacheEntry *)((flattenedClassCache) + 1)) + (index))
 #define J9_VM_FCC_ENTRY_FROM_CLASS(clazz, index) J9_VM_FCC_ENTRY_FROM_FCC((clazz)->flattenedClassCache, index)
+#define J9_VM_FCC_CLASS_FROM_ENTRY(entry) ((J9Class *)(~J9_VM_FCC_CLASS_FLAGS_MASK & (UDATA)(entry)->clazz))
+#define J9_VM_FCC_ENTRY_IS_STATIC_FIELD(entry) (J9_VM_FCC_CLASS_FLAGS_STATIC_FIELD == ((J9_VM_FCC_CLASS_FLAGS_MASK & ((UDATA)(entry)->clazz)) & J9_VM_FCC_CLASS_FLAGS_STATIC_FIELD))
```

Comment:
This could use the any/all bittest macros, and the inner mask is unnecessary (the static bit is necessarily a subset of all of the flag bits).